composer.json

```diff
@@ -12,30 +12,30 @@
         }
     ],
     "minimum-stability": "dev",
-    "prefer-stable": true,
     "require": {
         "php": ">=8.1",
         "doctrine/inflector": "^2.0",
         "nikic/php-parser": "^5.0",
-        "symfony/config": "^6.4|^7.0",
-        "symfony/console": "^6.4|^7.0",
-        "symfony/dependency-injection": "^6.4|^7.0",
+        "symfony/config": "^6.4|^7.0|^8.0",
+        "symfony/console": "^6.4|^7.0|^8.0",
+        "symfony/dependency-injection": "^6.4|^7.0|^8.0",
         "symfony/deprecation-contracts": "^2.2|^3",
-        "symfony/filesystem": "^6.4|^7.0",
-        "symfony/finder": "^6.4|^7.0",
-        "symfony/framework-bundle": "^6.4|^7.0",
-        "symfony/http-kernel": "^6.4|^7.0",
-        "symfony/process": "^6.4|^7.0"
+        "symfony/filesystem": "^6.4|^7.0|^8.0",
+        "symfony/finder": "^6.4|^7.0|^8.0",
+        "symfony/framework-bundle": "^6.4|^7.0|^8.0",
+        "symfony/http-kernel": "^6.4|^7.0|^8.0",
+        "symfony/process": "^6.4|^7.0|^8.0"
     },
     "require-dev": {
         "composer/semver": "^3.0",
-        "doctrine/doctrine-bundle": "^2.5.0",
+        "doctrine/doctrine-bundle": "^2.5.0|^3.0.0",
         "doctrine/orm": "^2.15|^3",
-        "symfony/http-client": "^6.4|^7.0",
-        "symfony/phpunit-bridge": "^6.4.1|^7.0",
-        "symfony/security-core": "^6.4|^7.0",
-        "symfony/security-http": "^6.4|^7.0",
-        "symfony/yaml": "^6.4|^7.0",
+        "doctrine/persistence": "^2.5.2|^3.0",
```

Comment:
I would suggest opening up doctrine/persistence to v4 too, as [doctrine bundle 3.0 requires v4 ](https://github.com/doctrine/DoctrineBundle/blob/3.0.x/composer.json#L35)

Reply:
Thanks!